### DISCUSSION TITLE
我在恐惧什么？ | /posts/fear-myself/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#578] [Announcements]
## 我在恐惧什么？

我不能停止思考，也不能停止厌恶这个世界，但是想想在残酷现实下仍有许多温情，于是也更加无法停止继续努力热爱这个世界。

https://yuanfan.rbind.io/posts/fear-myself/

**Comment by tianheg** [comment]
小花的答案对我很有用哦。即便不是朋友，我也想让对方知道我的真心、明白我的想法，但有时在对方对我不是太了解的情况下，很容易让对方退却，不再和我交流。

**Reply by earfanfan** [reply]
有用就好呀，我跟小花说她的答案对世界上另一个人很有用，哈哈，然而她记不得给我的答案是撒了……

>即便不是朋友，我也想让对方知道我的真心、明白我的想法

你竟然会有这样的想法，让我震惊之余，忍不住猜想你的年纪一定比我小很多。不知你有没有看过[鸿涛的博客](https://hongtaoh.com/cn/blog/)或者给他写信交流过，我上次钻牛角尖的时候有向他请教，他告诉我对待朋友也许可以冷淡一点，大意是说可以耐心地等着别人来找自己交流，这点我不大理解，不过可能对你来说会有用。

**Reply by tianheg** [reply]
> 忍不住猜想你的年纪一定比我小很多

00年出生的

---

我认识洪涛，以前我网站评论还在的时候跟他讨论过一些问题。

关于我说的：

> 即便不是朋友，我也想让对方知道我的真心、明白我的想法

我现在又思考一番，有两种想表达的想法：

1. 用真心换真心，我遇到陌生人，期待用真诚换取对方的信任
2. 对待喜欢的异性时，希望用真心打动对方

当然，能够这样做肯定是有条件的：

通过短暂的接触，我对某个陌生人产生了想要继续了解的兴趣，想让对方了解真实的自己。但是，让对方了解我到哪种程度是需要斟酌的。

**Reply by LFG12** [reply]
> 用真心换真心，我遇到陌生人，期待用真诚换取对方的信任
我也认同，把真实的自己显露出来，而不是装的自己，这样吸引来的就是对真实的自己感兴趣，而不是对装出来的自己感兴趣的人。如果人家不对真实的自己感兴趣，也不必浪费时间去讨好，相信茫茫人海中会有人看见。

又及，小哥看了下你的博客，发现咱们还有些相似之处，比如喜欢跑步，看你报名了香港马拉松，祝顺利完赛，我也喜欢跑步，跑过一次半马，对全马还是不敢挑战；还有一些其他的，比如比较关注健康，与人建立链接，还有和老人聊天哈哈。我下半年也是去深圳读书，还看你之前做过地图方面业务，有机会可以认识一下哈哈。

**Reply by tianheg** [reply]
我报名的是半马，全马现在我也是不太敢想。

好呀，你到深圳以后可以约个饭啥的。我博客首页有我的邮箱，到时候邮件联系。

**Comment by LFG12** [comment]
> 其三，任何一瞬的美丽都无法挽留，风景与我不能做到同时静止。
在火车上我的思维一般就很随意，没有目的的任思绪飘渺。
有一个奇妙的事情，对于火车中看到的景色给我留下了深深的印象，那应该是早期大学一次放假回家的途中，乘T181/182列车从武汉回聊城，还在湖北境内，我向着东边的方向望着窗外，一时间在群山之中，出现了一个小镇，小镇马路与火车垂直，主干路上有些红路灯，小镇处于一个平坦的洼地中，背后是群山环抱，同时时间是傍晚还生出了雾气，也有些灯光，交相辉映下那个小镇给我留下了深深的印象。当时只用眼睛记录了下来，那种模糊的印象和美一直停留在脑中，虽然遗憾只有那么一瞬间，没能用手机捕捉下来，但在脑海中一直保留下了这个景色。

与自己对话，真是一个神奇的能力啊。将内在划分为本我和自我意识，我感觉从来没有去划分过这两者，做事儿时一般也是统一的我就是我。当然，与自己对话这方面有时也很恍惚，有时要不知道自己是要什么。要是假如承认你这个的话，我的本我估计是一直被忽略之中很少表达，自我意识可能是要更多，这可能也是长久的“社会规训”和“家庭教育”下的产物。

稍微吐槽一下，钻牛角间的人我觉得是容易想的太多，要在自己内心里对一件事的考虑的很“精确”后才允许做反应。而我一般则觉得我的做事并没有那么的精准刻量。自己的处事方法是一种有以往各人的交往之中综合而成的一直处事方式（好听点叫融会贯通，不好听叫杂糅）。有时做一件事儿不好说出它的依据和考虑，但觉得该这么干，就这么做了。
所以考虑事情简单些，可能也感性凭经验一些。
这样大脑没那么累，想事情想得少。

**Reply by LFG12** [reply]
结合你文章的第一句话，我应该是一般没啥事时好像脑子就待机，就在停止思考，有什么契机需要思考再开动。
